iskolivery/app/views/users/index.html.erb

```diff
@@ -73,11 +74,14 @@
                          <td class = ""><%= user.verified %></td>
                          <td class = ""><%= form_tag verify_path do -%>
                                <%= hidden_field_tag :user_id, user.id %>
-                                   <%#= submit_tag 'Verify' %>
-
                                    <button class="ui mini toggle button" type="submit">Verify</button>
                          <% end -%>
                          </td>
+                         <td class = ""><%= form_tag verify_path do -%>
```

Comment:
This is for the disable action, but the form with a disable button is still submitting to `verify_path`
We don't have a named route for the disable action yet, but just a heads up to change this line when you run into a problem.